packages/api-client/src/types/event.types.d.ts

```diff
@@ -0,0 +1,11 @@
+export interface GetEventsRequest {
+  workspaceId: string
+  source: string
+  page?: number
+  limit?: number
+  sort?: string
+  order?: string
+  search?: string
+}
+
+export interface GetEventsResponse {}
```

Comment:
Update the response body. 

P.S. we want the response to be returned as `items: {...}[]`